es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super(width);
+    this.width = width;
+    this.area = this.width * this.width;
+    this.perimeter = this.width * 4;
```

Comment:
You should be passing this into `super` to create these properties, not creating them directly.